<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroPacienteWindow</class>
 <widget class="QWidget" name="CadastroPacienteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Paciente</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="text">
      <string>Cadastro de Paciente</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 16pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputIdentificacao">
     <property name="placeholderText">
      <string>Identificação do Paciente</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputProntuario">
     <property name="placeholderText">
      <string>Prontuário Eletrônico (Número)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoCadastrar">
     <property name="text">
      <string>Cadastrar Paciente</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoVoltar">
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>
